doc/start/quick-ceph-deploy.rst

```diff
@@ -289,6 +293,23 @@ For example::
    with multiple metadata servers.
 
 
+Add an RGW Instance
+-------------------
+
+To use the :term:`Ceph Object Gateway` component of Ceph, you must deploy an
+instance of :term:`RGW`.  Execute the following to create an new instance of
+RGW::
+
+    ceph-deploy rgw create {gateway-node}
+
+For example::
+
+    ceph-deploy rgw create node1
```

Comment:
can you mention that this will run the gateways on port 7480 by default.  to make it port 80, first all this to ceph.conf:

<pre>
[client]
  rgw frontends = civetweb port=80
</pre>
and to bind to an ipv6 address use
<pre>
  rgw frontends = civetweb port=[::]:80
</pre>